test/e2e/node_feature_discovery_test.go

```diff
@@ -133,7 +133,7 @@ func cleanupNode(cs clientset.Interface) {
 
 }
 
-func cleanupCRDs(cli *nfdclient.Clientset) {
+func cleanupCRDs(cli *nfdclient.Clientset, namespace string) {
```

Comment:
sorry for nitpicking but i think this func name should be `CRs` instead of `CRDs` especially when you pass `namespace` arg 😄 
```suggestion
func cleanupCRs(cli *nfdclient.Clientset, namespace string) {
```

Reply:
Yeah, that's more correct naming. Fixed

Reply:
:+1: